.openshift-ci/.pylintrc

```diff
@@ -277,7 +278,7 @@ indent-after-paren=4
 indent-string='    '
 
 # Maximum number of characters on a single line.
-max-line-length=100
+max-line-length=90
```

Comment:
Personally I'd advocate for 100 or even 120, as within these bounds horizontal space is still cheaper than vertical...

Reply:
120 seems reasonable to me.